crates/pera-core/src/dwallet_mpc/mpc_manager.rs

```diff
@@ -167,9 +171,35 @@ impl DWalletMPCManager {
                     );
                 }
             }
+            DWalletMPCChannelMessage::ValidatorDataForDKG(data) => {
+                if let Err(err) = self.handle_validator_data_for_dkg(data) {
+                    error!(
+                        "failed to handle validator data for DKG session with error: {:?}",
+                        err
+                    );
+                }
+            }
         }
     }
 
+    fn handle_validator_data_for_dkg(
+        &mut self,
+        data: ValidatorDataForDWalletSecretShare,
+    ) -> DwalletMPCResult<()> {
+        let epoch_store = self.epoch_store()?;
+        let party_id = authority_name_to_party_id(
+            &AuthorityPublicKeyBytes::from_bytes(&data.protocol_pubkey_bytes)
```

Comment:
Where do we verify that the sender who signed this message equals the pubkey that is written inside?